srcpkgs/coreboot-utils/template

```diff
@@ -0,0 +1,53 @@
+# Template file for 'coreboot-utils'
+pkgname=coreboot-utils
+version=4.8.1
+revision=1
+only_for_archs="i686 i686-musl x86_64 x86_64-musl"
```

Comment:
Intelmetool, inteltool, ifdtool doesn't make much sense on non-intel architectures, superiotool is irrevelant on anything other than i686* x86_64*. If you want I could make cbfstool and probably cbmem, ectool and nvramtool multiarch, but I don't have hardware to test them.

Reply:
Don't worry about it now. The package an be changed later on if someone has a need to use it on another architecture. I just wanted to ask since `only_for_archs` is used for the wrong purpose most of the time.